src/CMakeLists.txt

```diff
@@ -36,6 +36,16 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE LIBM2K_EXPORTS)
 if (ENABLE_LOG)
 	target_compile_definitions(${PROJECT_NAME} PUBLIC LIBM2K_ENABLE_LOG)
 endif()
+if (MSVC)
+	add_definitions(
+			/wd4250
```

Comment:
maybe add a comment for each of this, on what they do: `disable warning "abc"`

Reply:
Added.